web_src/css/install.css

```diff
@@ -18,7 +18,8 @@
   width: auto;
 }
 
-.page-content.install form.ui.form input {
+.page-content.install form.ui.form input:not([type="checkbox"]):not([type="radio"]),
```

Comment:
```suggestion
.page-content.install form.ui.form input:not([type="checkbox"],[type="radio"]),
```

Reply:
Had that before but reverted it because of unsure browser support in Palemoon, but I see it supports as of `v32.1.0 (2023-03-21)`, so we can go for it.